### PR TITLE
Updated unit test cases as per enhancements in UTC framework

### PR DESCRIPTION
**Pull request by cs-sameer-pathan**
#### Changes:
- Updated unit test cases as per Unit Test framework v2.0.0

#### Fix:
- 1096862: During Creation of Pull Request, user is not able to add reviews list.

#### Fix:
- [x] Tested and verified `Create Pull Request` action with and without reviewer.

### CODE REVIEW

File: azure-devops/tests/config_and_params.json [hunk 1]
```diff
@@ -205,7 +205,8 @@
         "password": "*******",
         "integer": -123,
         "json": {
-            "status": "val,mdnsue"
+            "status": "val,mdnsue",
+            "jhdbh": "kjwndcjksn"
```
Comment (cs-harsh-sharma) [inline azure-devops/tests/config_and_params.json]: The new framework has the invalid values defined in constants file
Please check if the value provided here is used in the respective test function.

File: azure-devops/tests/test_operations.py [hunk 2]
```diff
@@ -1,902 +1,695 @@
...
-    assert set(operations['list_pipeline_runs'](valid_credentials_with_token, input_params.copy()).keys()) == set(
-        schema.keys())
+def test_list_pipeline_runs_success(cache, valid_configuration_with_token, connector_details, params_json):
+    set_report_metadata(connector_details, "Get Pipeline Run List", "Verify with valid Input Parameters")
+    for result in run_success_test(cache, connector_details, operation_name='list_pipeline_runs',
+                                   action_params=params_json['list_pipeline_runs']):
+        assert result.get('status') == "Success"
+        print("\n\nlist_pipeline_runs result: {0}\n\n".format(result))
+        for pipeline_run in result.get('data', {}).get('value'):
+            logger.info("\n\nlist_pipeline_runs: {0}\n\n".format(pipeline_run))
```
Comment (cs-harsh-sharma) [inline azure-devops/tests/test_operations.py]: Do we intend to show these logs in test report. if not please remove

File: azure-devops/tests/config_and_params.json (hunk 1 above)
Comment (cs-sameer-pathan) [reply azure-devops/tests/config_and_params.json]: Removed

File: azure-devops/tests/test_operations.py (hunk 2 above)
Comment (cs-sameer-pathan) [reply azure-devops/tests/test_operations.py]: Removed

**Pull request merged**